smos/src/Smos/Terminal.hs

```diff
@@ -7,10 +7,10 @@ import Conduit
 import Data.Aeson
 import Data.ByteString (ByteString)
 import Data.Word
-import Graphics.Vty.Output.TerminfoBased (setWindowSize)
-import System.Posix
+-- import System.Posix
 import UnliftIO
 
+{-
```

Comment:
Same story here, it can be in `smos-e2e`.

Reply:
Fine for me.